content/fr/docs/concepts/configuration/secret.md

```diff
@@ -0,0 +1,980 @@
+---
+title: Secrets
+content_template: templates/concept
+feature:
+  title: Gestion du secret et de la configuration
+  description: >
+    Déployez et mettez à jour les secrets et la configuration des applications sans reconstruire votre image et sans dévoiler les secrets de la configuration de vos applications.
+weight: 50
+---
+
+
+{{% capture overview %}}
+
+Les objets `secrets` de Kubernetes vous permettent de stocker et de gérer des informations sensibles, telles que les mots de passe, les jetons OAuth et les clés ssh.
```

Comment:
nit:
```suggestion
Les objets `secret` de Kubernetes vous permettent de stocker et de gérer des informations sensibles, telles que les mots de passe, les jetons OAuth et les clés ssh.
```
The actual name for the object is Secret but the upstream localization (English) puts it in the wrong case&mdash;and in backticks, which is different from the recommended way to write API object names.